### PR TITLE
New security page

### PR DESCRIPTION
**Pull request by ottok**
A picture tells more than a thousand words..
image

### CODE REVIEW

File: lib/security-restrictions.php [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+          if ( isset($endpoints['/wp/v2/users']) ) {
+            unset($endpoints['/wp/v2/users']);
+          }
+          // Disable fetching a single user
+          if ( isset($endpoints['/wp/v2/users/(?P<id>[\d]+)']) ) {
+            unset($endpoints['/wp/v2/users/(?P<id>[\d]+)']);
+          }
+          return $endpoints;
+        }
+        add_filter('rest_endpoints', 'disable_user_endpoints', 1000);
```
Comment (JoosuaKoskinen) [inline lib/security-restrictions.php]: Should probably be:
`add_filter('rest_endpoints', 'Seravo\disable_user_endpoints', 1000);`

Currently the `disable_user_endpoints` function is not called at all and the error from it causes all the endpoints to break, not only `/wp/v2/users`.

Comment (ottok) [reply lib/security-restrictions.php]: Thanks for noting! I did not test the site with wp-cli at all.

Namespacing is now fixed.

**Comment by ottok** [comment]
@JoosuaKoskinen OK now? Will you finish the review with merge?

**Comment by JoosuaKoskinen** [comment]
> @JoosuaKoskinen OK now? Will you finish the review with merge?

Yep, seems to be working now.

**Pull request merged**